pkg/subctl/cmd/createcrds.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+)
+
+func createEndpointsCRD() *apiextensions.CustomResourceDefinition {
+	// Define endpoints.submariner.io CRD spec
+	endpoints_crd_spec_names := apiextensions.CustomResourceDefinitionNames{Plural: "endpoints", Singular: "endpoint", ListKind: "EndpointList", Kind: "Endpoint"}
+	endpoints_crd_spec_versions := apiextensions.CustomResourceDefinitionVersion{Name: "v1", Served: true, Storage: true}
+	endpoints_crd_spec_conversion := apiextensions.CustomResourceConversion{Strategy: "None"}
+	endpoints_crd_spec := apiextensions.CustomResourceDefinitionSpec{Group: "submariner.io", Names: endpoints_crd_spec_names, Scope: "Namespaced", Versions: []apiextensions.CustomResourceDefinitionVersion{endpoints_crd_spec_versions}, Version: "v1", Conversion: &endpoints_crd_spec_conversion}
+
+	// Define endpoints.submariner.io CRD status
+	endpoints_crd_status_names := apiextensions.CustomResourceDefinitionNames{Plural: "endpoints", Singular: "endpoint", ListKind: "EndpointList", Kind: "Endpoint"}
+	endpoints_crd_status_storedversions := []string{"v1"}
+	endpoints_crd_status:= apiextensions.CustomResourceDefinitionStatus{AcceptedNames: endpoints_crd_status_names, StoredVersions: endpoints_crd_status_storedversions}
```

Comment:
As I understand it, status shouldn’t be provided on CRD creation, it’s filled in by K8s as the CRD status changes.